python/nav/web/templates/styleguide.html

```diff
@@ -411,8 +411,8 @@ <h3 class="underline">Forms</h3>
       </ul>
 
       <p>
-        There are many customizations done to the crispy forms. Basically, if you
-        need to create forms - ask on mail/jabber/irc.
+        Use existing forms as examples for how to build your own.  If you have questions, ask them
```

Comment:
```suggestion
        Use existing forms as examples for how to build your own. If you have questions, ask them
```

Reply:
Wow, I'm sorry my old-school ingrained habits annoy you so ;-)

Reply:
Oh, I didn't know this was on purpose, I just thought you spent a bit too long on the space bar :grin: